Clean Swift/Models.xctemplate/___FILEBASENAME___Models.swift

```diff
@@ -8,11 +8,18 @@
 
 import UIKit
 
-struct ___FILEBASENAMEASIDENTIFIER___Request {
-}
-
-struct ___FILEBASENAMEASIDENTIFIER___Response {
-}
-
-struct ___FILEBASENAMEASIDENTIFIER___ViewModel {
+struct ___FILEBASENAMEASIDENTIFIER___
+{
+    struct Something
```

Comment:
`Something` refers to naming used on other places (`doSomething`, `displaySomething` etc.). Also it's acording to latest version of clean-swift templates. We've been trying to use the nesting (kind of namespacing) and it helps us make it more clear which actions are cycling through VIP...but it could be only a matter of opinion.